plugins/sudoers/iolog_client.c

```diff
@@ -234,12 +234,15 @@ tls_init(struct client_closure *closure)
             closure->log_details->key_file = closure->log_details->cert_file;
         }
         if (!SSL_CTX_use_PrivateKey_file(closure->ssl_ctx,
-                closure->log_details->key_file, X509_FILETYPE_PEM)) {
+                closure->log_details->key_file, SSL_FILETYPE_PEM) ||
+                !SSL_CTX_check_private_key(closure->ssl_ctx)) {
             errstr = ERR_reason_error_string(ERR_get_error());
             sudo_warnx(U_("Unable to load private key into the ssl context: %s"),
                 errstr);
             goto bad;
         }
+        if (!SSL_CTX_check_private_key(closure->ssl_ctx)) {
```

Comment:
I suspect this is by mistake (case already handled above).

Reply:
Fixed.